plotly/plotly/plotly.py

```diff
@@ -150,6 +204,11 @@ def plot(figure_or_data, validate=True, **plot_options):
     auto_open (default=True) -- Toggle browser options
         True: open this plot in a new browser tab
         False: do not open plot in the browser, but do return the unique url
+    sharing ('public' | 'private' | 'secret')
+        'public': set the privacy of the plot to public
+        'private': set the privacy of the plot as private
```

Comment:
```
sharing ('public' | 'private' | 'sharing') -- Toggle who can view this graph
- `'public': Anyone can view this graph. It will appear in your profile and can appear in search engines. You do not need to be logged in to Plotly to view this chart.
- `'private': Only you can view and edit this plot. It will not appear in the Plotly feed, your profile, or search engines. You must be logged in to Plotly to view this graph. You can privately share this graph with other Plotly users in your online Plotly account and they will need to be logged in to view this plot.
- 'secret': Anyone with this secret link can view this chart. It will not appear in the Plotly feed, your profile, or search engines. If it is embedded inside a webpage or an IPython notebook, anybody who is viewing that page will be able to view the graph. You do not need to be logged in to view this plot.